compiler/parser/RdrHsSyn.hs

```diff
@@ -3671,9 +3671,12 @@ mkInterfaceInstanceDecls parentName sharedBinds (L loc interfaceInstance) = do
                     (ValBinds noExt
                       (unitBag
                         (noLoc
-                          (FunBind noExt shortViewName viimdMatches WpHole [])))
+                          (FunBind noExt internalViewName
+                            (renameMatchGroup internalViewName viimdMatches)
```

Comment:
this makes it so all the bindings are for `$view` instead of `view`, leaving `(DA.Internal.Desugar.)view` visible